diff --git a/dist/modules/globalifySelector.js b/dist/modules/globalifySelector.js
index fff3bd6c406d365a697a684ff245cc33d724285b..e1c72a965651748d122aeac7a098022a68ad8467 100644
--- a/dist/modules/globalifySelector.js
+++ b/dist/modules/globalifySelector.js
@@ -9,7 +9,7 @@ exports.globalifySelector = void 0;
  * escaped combinators like `\~`.
  */
 // TODO: maybe replace this ugly pattern with an actual selector parser? (https://github.com/leaverou/parsel, 2kb)
-const combinatorPattern = /(?<!\\)(?:\\\\)*([ >+~,]\s*)(?![^[]+\]|\d)/g;
+const combinatorPattern = /(?<!\\)(?:\\\\)*([ >+~,]\s*)(?![^(]*\))(?![^[]+\]|\d)/g;
 function globalifySelector(selector) {
     const parts = selector.trim().split(combinatorPattern);
     const newSelector = [];
